_includes/transaction-response.md

```diff
@@ -100,7 +100,7 @@ Content-Type: application/json
 | └─➔&nbsp;`number`                 | `string`  | The transaction `number`, useful when there's need to reference the transaction in human communication. Not usable for programmatic identification of the transaction, for that `id` should be used instead. |
 | └─➔&nbsp;`amount`                 | `integer` | {% include field-description-amount.md %}                                                                                                                                                                    |
 | └─➔&nbsp;`vatAmount`              | `integer` | {% include field-description-vatamount.md %}                                                           |
-| └─➔&nbsp;`description`            | `string`  | A human readable description of maximum 40 characters of the transaction.                                                                                                                                    |
+| └─➔&nbsp;`description`            | `string`  | {% include field-description-description.md payment_instrument=payment_instrument %}                                                                                                                                    |
```

Comment:
In this include, the variable name is called `instrument`, not `payment_instrument`. I suggest fixing it to be `payment_instrument` everywhere.